packs/ac-feats/Billowing_Wings_p83NzvyOMMa5Dgdh.json

```
@@ -0,0 +1,94 @@
+{
+  "name": "Billowing Wings",
+  "type": "feat",
+  "effects": [],
+  "system": {
+    "description": {
+      "gm": "",
+      "value": "<p>You’ve trained your winged companions to gather air in their wings, whipping up a concentrated gust of wind at opponents, potentially staggering them.</p><p>Your companions with wings gain a gust ranged unarmed attack, which deals @Damage[1d4[bludgeoning]] damage with a range of 30 feet and has the air and propulsive traits. On a critical hit, the target is pushed back 5 feet. This is forced movement.</p>"
```

```suggestion
      "value": "<p>You've trained your winged companions to gather air in their wings, whipping up a concentrated gust of wind at opponents, potentially staggering them.</p><p>Your companions with wings gain a gust ranged unarmed attack, which deals 1d4 bludgeoning damage with a range of 30 feet and has the air and propulsive traits. On a critical hit, the target is pushed back 5 feet. This is forced movement.</p>"
```
We follow the same style guide as the system does, which means no rsquo characters (the fancy apostrophe), and no inline damage for things that are automated.